templates/monitoring/schema_error_counts_v1/view.sql

```diff
@@ -0,0 +1,52 @@
+CREATE
+OR REPLACE VIEW `moz-fx-data-shared-prod.monitoring.schema_error_counts_v1` AS WITH extracted AS (
+  SELECT
+    TIMESTAMP_TRUNC(submission_timestamp, HOUR) AS hour,
+    job_name,
+    document_namespace,
+    document_type,
+    error_message
+  FROM
+    `moz-fx-data-shared-prod.payload_bytes_error.*`
+  WHERE
+    submission_timestamp < TIMESTAMP_TRUNC(current_timestamp, day)
+    AND submission_timestamp > TIMESTAMP_SUB(
+      TIMESTAMP_TRUNC(current_timestamp, day),
+      INTERVAL 28 * 24 hour
+    )
+    AND exception_class = 'org.everit.json.schema.ValidationException'
+),
+count_errors AS (
+  SELECT
+    document_namespace,
+    document_type,
+    hour,
+    job_name,
+    SPLIT(error_message, ":")[OFFSET (1)] AS path,
+    COUNT(*) AS error_count,
+    ROW_NUMBER() OVER (
+      PARTITION BY
+        hour,
+        document_namespace,
+        document_type
+      ORDER BY
+        COUNT(*) DESC
+    ) AS error_rank
```

Comment:
I could have sworn you aren't allowed to use window functions in conjunction with a group by statement. please confirm this actually works before merging.

Reply:
Yep, I've been building this in the BigQuery console, so this should work.